Libraries/Lists/VirtualizedSectionList.js

```diff
@@ -19,35 +19,11 @@ import type {ViewToken} from 'ViewabilityHelper';
 import type {Props as VirtualizedListProps} from 'VirtualizedList';
 
 type Item = any;
-type SectionItem = any;
 
-type SectionBase = {
-  // Must be provided directly on each section.
-  data: $ReadOnlyArray<SectionItem>,
-  key?: string,
-
-  // Optional props will override list-wide props just for this section.
-  renderItem?: ?({
-    item: SectionItem,
-    index: number,
-    section: SectionBase,
-    separators: {
-      highlight: () => void,
-      unhighlight: () => void,
-      updateProps: (select: 'leading' | 'trailing', newProps: Object) => void,
-    },
-  }) => ?React.Element<any>,
-  ItemSeparatorComponent?: ?React.ComponentType<any>,
-  keyExtractor?: (item: SectionItem, index: ?number) => string,
-
-  // TODO: support more optional/override props
-  // FooterComponent?: ?ReactClass<any>,
-  // HeaderComponent?: ?ReactClass<any>,
-  // onViewableItemsChanged?: ({viewableItems: Array<ViewToken>, changed: Array<ViewToken>}) => void,
-};
+type SectionBase = Object;
```

Comment:
This is a regression. Could you work on putting back the proper Flow Type and running flow on the react-native repo to verify it?

Reply:
The SectionBase type can't be specified, because it can handle different type of datas, like classic js object or Immutable object, by exemple the data type of `VirtualizedList` is `any`.
For `SectionList` component `SectionBase` can be specified because we know the data must be javascript object.